diff -uNr wireshark-1.0.7/Makefile.am wireshark-1.0.7-btbb/Makefile.am
--- wireshark-1.0.7/Makefile.am	2009-05-04 22:09:15.000000000 -0600
+++ wireshark-1.0.7-btbb/Makefile.am	2009-04-08 17:33:06.000000000 -0600
@@ -241,7 +241,6 @@
 	-dlopen plugins/agentx/agentx.la \
 	-dlopen plugins/artnet/artnet.la \
 	-dlopen plugins/asn1/asn1.la \
-	-dlopen plugins/btbb/btbb.la \
 	-dlopen plugins/ciscosm/ciscosm.la \
 	-dlopen plugins/docsis/docsis.la \
 	-dlopen plugins/enttec/enttec.la \
diff -uNr wireshark-1.0.7/Makefile.nmake wireshark-1.0.7-btbb/Makefile.nmake
--- wireshark-1.0.7/Makefile.nmake	2009-05-04 22:10:06.000000000 -0600
+++ wireshark-1.0.7-btbb/Makefile.nmake	2009-04-08 17:33:07.000000000 -0600
@@ -1003,7 +1003,6 @@
 	xcopy ".\plugins\agentx\agentx.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy ".\plugins\artnet\artnet.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy ".\plugins\asn1\asn1.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
-	xcopy ".\plugins\btbb\btbb.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy ".\plugins\ciscosm\ciscosm.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy ".\plugins\docsis\docsis.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy ".\plugins\enttec\enttec.dll" $(INSTALL_DIR)\plugins\$(VERSION) /d
diff -uNr wireshark-1.0.7/configure.in wireshark-1.0.7-btbb/configure.in
--- wireshark-1.0.7/configure.in	2009-05-04 22:09:47.000000000 -0600
+++ wireshark-1.0.7-btbb/configure.in	2009-04-08 17:33:06.000000000 -0600
@@ -1699,7 +1699,6 @@
   plugins/agentx/Makefile
   plugins/artnet/Makefile
   plugins/asn1/Makefile
-  plugins/btbb/Makefile
   plugins/ciscosm/Makefile
   plugins/docsis/Makefile
   plugins/enttec/Makefile
diff -uNr wireshark-1.0.7/packaging/nsis/Makefile.nmake wireshark-1.0.7-btbb/packaging/nsis/Makefile.nmake
--- wireshark-1.0.7/packaging/nsis/Makefile.nmake	2009-05-04 22:11:46.000000000 -0600
+++ wireshark-1.0.7-btbb/packaging/nsis/Makefile.nmake	2009-04-08 17:33:02.000000000 -0600
@@ -71,7 +71,6 @@
 	../../plugins/agentx/agentx.dll \
 	../../plugins/artnet/artnet.dll \
 	../../plugins/asn1/asn1.dll \
-	../../plugins/btbb/btbb.dll \
 	../../plugins/ciscosm/ciscosm.dll \
 	../../plugins/docsis/docsis.dll \
 	../../plugins/enttec/enttec.dll \
diff -uNr wireshark-1.0.7/packaging/nsis/wireshark.nsi wireshark-1.0.7-btbb/packaging/nsis/wireshark.nsi
--- wireshark-1.0.7/packaging/nsis/wireshark.nsi	2009-05-04 22:12:01.000000000 -0600
+++ wireshark-1.0.7-btbb/packaging/nsis/wireshark.nsi	2009-04-08 17:33:02.000000000 -0600
@@ -835,7 +835,6 @@
 File "..\..\plugins\agentx\agentx.dll"
 File "..\..\plugins\artnet\artnet.dll"
 File "..\..\plugins\asn1\asn1.dll"
-File "..\..\plugins\btbb\btbb.dll"
 File "..\..\plugins\ciscosm\ciscosm.dll"
 File "..\..\plugins\docsis\docsis.dll"
 File "..\..\plugins\enttec\enttec.dll"
diff -uNr wireshark-1.0.7/plugins/Makefile.am wireshark-1.0.7-btbb/plugins/Makefile.am
--- wireshark-1.0.7/plugins/Makefile.am	2009-05-04 22:12:12.000000000 -0600
+++ wireshark-1.0.7-btbb/plugins/Makefile.am	2009-04-08 17:32:04.000000000 -0600
@@ -25,7 +25,6 @@
 	agentx \
 	artnet \
 	asn1 \
-	btbb \
 	ciscosm \
 	docsis \
 	enttec \
diff -uNr wireshark-1.0.7/plugins/Makefile.nmake wireshark-1.0.7-btbb/plugins/Makefile.nmake
--- wireshark-1.0.7/plugins/Makefile.nmake	2009-05-04 22:12:43.000000000 -0600
+++ wireshark-1.0.7-btbb/plugins/Makefile.nmake	2009-04-08 17:32:04.000000000 -0600
@@ -38,9 +38,6 @@
 	cd asn1
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
-	cd btbb
-	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
-	cd ..
 	cd ciscosm
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
@@ -127,7 +124,6 @@
 	xcopy agentx\*.dll $(VERSION) /d
 	xcopy artnet\*.dll $(VERSION) /d
 	xcopy asn1\*.dll $(VERSION) /d
-	xcopy btbb\*.dll $(VERSION) /d
 	xcopy ciscosm\*.dll $(VERSION) /d
 	xcopy docsis\*.dll $(VERSION) /d
 	xcopy enttec\*.dll $(VERSION) /d
