diff -uNr wireshark-1.2.0/configure.in wireshark-1.2.0-btbb/configure.in
--- wireshark-1.2.0/configure.in	2009-06-16 05:04:01.000000000 +0100
+++ wireshark-1.2.0-btbb/configure.in	2009-06-21 13:36:49.000000000 +0100
@@ -1714,6 +1714,7 @@
   packaging/svr4/pkginfo
   plugins/Makefile
   plugins/asn1/Makefile
+  plugins/btbb/Makefile
   plugins/docsis/Makefile
   plugins/ethercat/Makefile
   plugins/giop/Makefile
diff -uNr wireshark-1.2.0/Makefile.am wireshark-1.2.0-btbb/Makefile.am
--- wireshark-1.2.0/Makefile.am	2009-06-16 05:04:01.000000000 +0100
+++ wireshark-1.2.0-btbb/Makefile.am	2009-06-21 13:34:40.000000000 +0100
@@ -244,6 +244,7 @@
 -include plugins/Custom.make
 plugin_ldadd = $(_CUSTOM_plugin_ldadd_) \
 	-dlopen plugins/asn1/asn1.la \
+	-dlopen plugins/btbb/btbb.la \
 	-dlopen plugins/docsis/docsis.la \
 	-dlopen plugins/ethercat/ethercat.la \
 	-dlopen plugins/giop/cosnaming.la \
diff -uNr wireshark-1.2.0/packaging/nsis/Makefile.nmake wireshark-1.2.0-btbb/packaging/nsis/Makefile.nmake
--- wireshark-1.2.0/packaging/nsis/Makefile.nmake	2009-06-16 05:03:53.000000000 +0100
+++ wireshark-1.2.0-btbb/packaging/nsis/Makefile.nmake	2009-06-21 13:38:15.000000000 +0100
@@ -44,6 +44,7 @@
 
 PLUGINS= \
 	../../plugins/asn1/asn1.dll \
+	../../plugins/btbb/btbb.dll \
 	../../plugins/docsis/docsis.dll \
 	../../plugins/ethercat/ethercat.dll \
 	../../plugins/giop/coseventcomm.dll \
diff -uNr wireshark-1.2.0/packaging/nsis/wireshark.nsi wireshark-1.2.0-btbb/packaging/nsis/wireshark.nsi
--- wireshark-1.2.0/packaging/nsis/wireshark.nsi	2009-06-16 05:03:53.000000000 +0100
+++ wireshark-1.2.0-btbb/packaging/nsis/wireshark.nsi	2009-06-21 13:38:49.000000000 +0100
@@ -809,6 +809,7 @@
 ;-------------------------------------------
 SetOutPath '$INSTDIR\plugins\${VERSION}'
 File "..\..\plugins\asn1\asn1.dll"
+File "..\..\plugins\btbb\btbb.dll"
 File "..\..\plugins\docsis\docsis.dll"
 File "..\..\plugins\ethercat\ethercat.dll"
 File "..\..\plugins\giop\coseventcomm.dll"
diff -uNr wireshark-1.2.0/plugins/Makefile.am wireshark-1.2.0-btbb/plugins/Makefile.am
--- wireshark-1.2.0/plugins/Makefile.am	2009-06-16 05:01:29.000000000 +0100
+++ wireshark-1.2.0-btbb/plugins/Makefile.am	2009-06-21 13:39:43.000000000 +0100
@@ -24,6 +24,7 @@
 -include Custom.make
 SUBDIRS = $(_CUSTOM_SUBDIRS_) \
 	asn1 \
+	btbb \
 	docsis \
 	ethercat \
 	giop \
diff -uNr wireshark-1.2.0/plugins/Makefile.nmake wireshark-1.2.0-btbb/plugins/Makefile.nmake
--- wireshark-1.2.0/plugins/Makefile.nmake	2009-06-16 05:01:29.000000000 +0100
+++ wireshark-1.2.0-btbb/plugins/Makefile.nmake	2009-06-21 13:56:42.000000000 +0100
@@ -33,6 +33,9 @@
 	cd asn1
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
+	cd btbb
+	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
+	cd ..
 	cd docsis
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
@@ -86,6 +89,7 @@
 !IFDEF ENABLE_LIBWIRESHARK
 	cd..
 	xcopy plugins\asn1\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
+	xcopy plugins\btbb\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\docsis\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\ethercat\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\giop\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
